test/sql/test_types.py

```diff
@@ -3809,6 +3809,11 @@ def test_nonmutable_comparison(self):
         ):
             assert p1.compare_values(p1.copy_value(obj), obj)
 
+    def test_customized_impl(self):
+        p1 = PickleType(impl=mysql.LONGBLOB)
+
+        assert isinstance(p1.impl, mysql.LONGBLOB)
```

Comment:
let's also add an assert on a normal `PickleType` to verify it's still largebinary. something like `assert isinstance(PickleType().impl, LargeBinary)`

Reply:
done